pkg/controller/controller_manager.go

```diff
@@ -0,0 +1,75 @@
+// +build azurediskv2
```

Comment:
`manager.go` should be sufficient for the file name.

Reply:
I am a little wary of this since manager.go would be very generic. changed the name anyway.